.github/workflows/build.yml

```
@@ -8,9 +8,12 @@ on:
     types: [apb]
 
 env:
+  BRANCH_NAME: improvement/support_atx_closed_markdown_headers
   CURL_CACHE_DIR: ~/.cache/curl
+  DEPTH: 1
   PIP_CACHE_DIR: ~/.cache/pip
   PRE_COMMIT_CACHE_DIR: ~/.cache/pre-commit
+  REPO_URL: https://github.com/mcdonnnj/terraform-docs.git
   RUN_TMATE: ${{ secrets.RUN_TMATE }}
```

I'd prefer these prepended to make it clear how they're being used.

```suggestion
  CURL_CACHE_DIR: ~/.cache/curl
  PIP_CACHE_DIR: ~/.cache/pip
  PRE_COMMIT_CACHE_DIR: ~/.cache/pre-commit
  RUN_TMATE: ${{ secrets.RUN_TMATE }}
  TERRAFORM_DOCS_BRANCH_NAME: improvement/support_atx_closed_markdown_headers
  TERRAFORM_DOCS_REPO_DEPTH: 1
  TERRAFORM_DOCS_REPO_URL: https://github.com/mcdonnnj/terraform-docs.git
```